app/controllers/users_controller.rb

```diff
@@ -36,7 +43,7 @@ def create
     @user.setup_and_save_user
 
     if @user.persisted?
-      @user.tune_after_persisted(current_organization)
+      @user.tune_after_persisted(current_organization, get_tags)
```

Comment:
instead of passing around this tags variable in different methods, I'd try to use a different approach, some ideas:
- send the `tag_list` attribute (provided by teh Taggable concern) + nested attributes
- use a new method to update tags in all places the same way, ala `@user.add_tags(organization)`

Reply:
I'm not quite sure how to do the first one, so if they are just as efficient I'll do the second one.